crates/composefs/src/repository.rs

```diff
@@ -135,7 +135,7 @@ impl<ObjectID: FsVerityHashValue> Repository<ObjectID> {
                     Err(CompareVerityError::Measure(MeasureVerityError::VerityMissing))
                         if self.insecure =>
                     {
-                        match enable_verity::<ObjectID>(&fd) {
+                        match enable_verity_raw::<ObjectID>(&fd) {
```

Comment:
Note I very intentionally left this as the `_raw` variant for now so it's effectively a no-op as far as `Repository` is concerned, but should probably be updated to use one of the new APIs.